utilities/rocAL/rocAL_dataloader/rocAL_dataloader.cpp

```diff
@@ -267,14 +245,12 @@ int main(int argc, const char ** argv)
                     }
                 }
             }
-            cv::imshow("output",mat_output);
+            cv::imwrite("output_for_cifar.png",mat_output);
         }
         else
         {
-            mat_input.copyTo(mat_output(cv::Rect(  col_counter*w, 0, w, h)));
-            cv::imshow("output",mat_output);
```

Comment:
Why imshow removed across the files?

Reply:
imshow does not work in dockers, hence was removed for testing, will add it back

Reply:
done - added it back